serializers/src/glTF/2.0/babylon.glTFExporter.ts

```diff
@@ -937,13 +944,36 @@ module BABYLON.GLTF2 {
                 }
 
                 if (bufferMesh.subMeshes) {
-                    uvCoordsPresent = false;
                     // go through all mesh primitives (submeshes)
                     for (const submesh of bufferMesh.subMeshes) {
+                        uvCoordsPresent = false;
+                        let babylonMaterial = submesh.getMaterial();
+
+                        let materialIndex: Nullable<number> = null;
+                        if (babylonMaterial) {
+                            if (babylonMaterial instanceof MultiMaterial) {
+                                const babylonMultiMaterial = babylonMaterial as MultiMaterial;

```

Comment:
this isn't necessary as typescript automatically makes babylonMaterial a MultiMaterial when you pass the if check

Reply:
Thanks, that's good to know